packages/fx-core/src/plugins/resource/frontend/v3/index.ts

```diff
@@ -113,7 +110,7 @@ export class NodeJSTabFrontendPlugin implements v3.PluginV3 {
         }
       },
     });
-    await progress.end(true);
+    await ProgressHelper.endProgress(true);
```

Comment:
It looks the PR is not only related to customize install but involve refactor of the current code, right?

Reply:
Yes. I did this refactor because I want to unify plugin and v3 plugin, rather than maintain these duplicated code. But soon I found they are based on different error handling, so I stop at refactoring the progress helper.